pelican/__init__.py

```diff
@@ -538,9 +540,9 @@ def main(argv=None):
     except KeyboardInterrupt:
         logger.warning('Keyboard interrupt received. Exiting.')
     except Exception as e:
-        logger.critical('%s', e)
+        logger.critical("%s: %s" % (e.__class__.__name__, e))
```

Comment:
Don't pre-format logs and let the logger handle them please :)
```python
logger.critical("%s: %s", e.__class__.__name__, e)
```

Reply:
This code will give you:
```
[21:27:20] CRITICAL ZeroDivisionError: division by zero
```

the old code will give you:
```
[21:29:08] CRITICAL division by zero
```

Ideally the logger should handle the formatting, but I feel like the default is failing us here as it looses the "type" of error that is being throw.

Are you certain you want to hide the error "type"?

Reply:
I am not hiding it? Difference between my suggestion and your is that, you pre-format the message using type and error text (`%` operator) before sending it to logger, mine sends those as arguments to the logger (`,` in between string and arguments).

Reply:
bc21922cf217c4e05f9c28fbc30349271735798e resolves this. I'd missed the `,` vs `%`...